google/cloud/forseti/services/scanner/service.py

```diff
@@ -82,15 +82,36 @@ def Run(self, _, context):
         progress_queue = Queue()
 
         model_name = self._get_handle(context)
-        LOGGER.info('Run scanner service with model: %s', model_name)
-        self.service_config.run_in_background(
-            lambda: self.scanner.run(model_name,
-                                     progress_queue,
-                                     self.service_config))
+        if not model_name:
+            progress_queue.put(
+                'You must specify a model before running the Forseti'
+                ' scanner. Run `forseti model -h` for more information.')
+            progress_queue.put(None)
+        else:
+            LOGGER.info('Run scanner service with model: %s', model_name)
+            self.service_config.run_in_background(
+                lambda: self._run_scanner(model_name, progress_queue))
 
         for progress_message in iter(progress_queue.get, None):
             yield scanner_pb2.Progress(message=progress_message)
 
+    def _run_scanner(self, model_name, progress_queue):
+        """Run scanner.
+
+        Args:
+            model_name (str): Model name.
+            progress_queue (Queue): Progress queue.
+        """
+        # pylint: disable=broad-except
```

Comment:
Move this pylint disable to the same line as the except, so that this is not propagated down through this whole file.  Fix above too.

```
except Exception as e:  # pylint: disable=broad-except
```

Reply:
Updated.